lib/pages/athlete/AthletePage.dart

```diff
@@ -886,18 +886,18 @@ class _AthletePageState extends State<AthletePage> {
                             });
                           },
                           child: Icon(Icons.arrow_back,
-                              size: 50, color: Colors.white))),
+                              size: 20, color: Colors.white))),
                   // APT Icon
                   Container(
-                    width: 30,
+                    width: 10,
                   ),
                   // Player Name
                   Container(
                       child: Text(athlete.name,
                           style: textStyle(Colors.white, 28, false, false))),
                   // '|' Symbol
                   Container(
-                      width: 50,
+                      width: 10,
```

Comment:
In my opinion, this will make the contents between the line break to look like its too close to each other. Increase the width a little to give more breathing room for the other widgets